tests/jobs/base.bull.job.test.js

```diff
@@ -0,0 +1,31 @@
+const BaseJob = require('../../jobs/base.bull.job');
+
+describe('BaseHandler', () => {
+	beforeAll(() => {
+	})
+	describe('settings check', () => {
+		test('should have default setting values', () => {
+			const baseJob = new BaseJob('test');
+			expect(baseJob.queueOpts.settings.lockDuration).toEqual(60000);
+			expect(baseJob.queueOpts.settings.lockRenewTime).toEqual(60000 / 4);
```

Comment:
hm but if this test is passing, then I'm wrong, so not sure what I'm missing

Reply:
oh i get it. if `overrides.settings` exist but doesn't have the `lockDuration` key, then that will be `undefined`, right?